providers/os/resources/packages/windows_packages.go

```diff
@@ -245,13 +245,58 @@ func (w *WinPkgManager) getInstalledApps() ([]Package, error) {
 		return w.getLocalInstalledApps()
 	}
 
+	if w.conn.Type() == shared.Type_FileSystem {
+		return w.getFsInstalledApps()
+	}
+
 	cmd, err := w.conn.RunCommand(powershell.Wrap(installedAppsScript))
 	if err != nil {
 		return nil, fmt.Errorf("could not read app package list")
 	}
 	return ParseWindowsAppPackages(cmd.Stdout)
 }
 
+func (w *WinPkgManager) getFsInstalledApps() ([]Package, error) {
+	rh := registry.NewRegistryHandler()
+	defer func() {
+		err := rh.UnloadSubkeys()
+		if err != nil {
+			log.Debug().Err(err).Msg("could not unload registry subkeys")
+		}
+	}()
+	fi, err := w.conn.FileInfo(registry.SoftwareRegPath)
+	if err != nil {
+		log.Debug().Err(err).Msg("could not find SOFTWARE registry key file")
+		return nil, err
+	}
+	err = rh.LoadSubkey(registry.Software, fi.Path)
+	if err != nil {
+		log.Debug().Err(err).Msg("could not load SOFTWARE registry key file")
+		return nil, err
+	}
+	pkgs := []string{
+		"Microsoft\\Windows\\CurrentVersion\\Uninstall",
+		"Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall",
+	}
+	packages := []Package{}
+	for _, r := range pkgs {
+		// we get all the packages, found under the pkgs paths
+		children, err := rh.GetNativeRegistryKeyChildren(registry.Software, r)
+		if err != nil {
+			continue
+		}
+		for _, c := range children {
+			// for each package the information is contained as children of that registry's key
+			p := w.getPackageFromRegistryKey(c)
```

Comment:
use the same logic as the `localInstalledApps`, only difference is we let the registry handler tell us where the keys are mounted via `GetNativeRegistryKeyChildren`